assembly/src/ast/mod.rs

```diff
@@ -717,12 +717,17 @@ impl Deserializable for ProcedureAst {
 pub struct ProcReExport {
     proc_id: ProcedureId,
     name: ProcedureName,
+    docs: Option<String>,
 }
 
 impl ProcReExport {
     /// Creates a new re-exported procedure.
-    pub fn new(proc_id: ProcedureId, name: ProcedureName) -> Self {
-        Self { proc_id, name }
+    pub fn new(proc_id: ProcedureId, name: ProcedureName, docs: Option<String>) -> Self {
```

Comment:
I think in this case since we have docs as `Option` from `take_doc_comment_at` and we only use it once, it's probably cleaner to just pass it to the `new` constructor but I can change it if you feel strongly about it.

Reply:
No, leave it as it is - it's just a nit.